### DISCUSSION TITLE
Can I play with more than 10 players at same time?

### DISCUSSION THREAD

**Discussion by deccosantos** [Impostor/Impostor#245] [General]
I really wanna know, before create my on server if I can set more than 10 player to play with me at the same time? For example, I saw an youtuber that plays with 100 players at the same time in a custom server, would it be possible with this server?

**Comment by danbudworthmead** [comment]
No

**Reply by deccosantos** [reply]
> No
> […](#)

ok thanks

**Reply by nailuj29** [reply]
> 
> No


Is there a way modifying the source code can allow that?

**Reply by TheChilliPL** [reply]
Bump—I'd be interested in such an option to play with at least a couple more players than 10

**Reply by le-show-de-didoub74** [reply]
It's possible with a mods (plugin) : [this](https://github.com/CrowdedMods/CrowdedMod) mods can do what you want juste put it in the plugin folder and install BepInEx
